.pre-commit-hooks.yaml

```diff
@@ -4,7 +4,7 @@
   description: Runs hadolint Docker image to lint Dockerfiles
   language: docker_image
   types: ["dockerfile"]
-  entry: hadolint/hadolint hadolint
+  entry: hadolint/hadolint:v2.3.0 hadolint
```

Comment:
The only downside is that, as a developer, you will need to bump this as well when updating version.